lib/streamlit/DeltaGenerator.py

```diff
@@ -2657,11 +2671,13 @@ def deck_gl_chart(self, element, spec=None, **kwargs):
         if not suppress_deprecation_warning:
             import streamlit as st
 
-            st.warning("""
+            st.warning(
```

Comment:
Are these whitespace changes due to a difference in `black` between your setup and someone else's, or what? Just curious, since they seem spurious.

Reply:
I think this may be due to either the previous author forgetting to run `black` (PyCharm doesn't seem to run git-hooks properly - at least, not on my machine, so I have to remember to do `make pyformat` before committing), OR possibly due to a merge happening on Github and therefore no hooks running?

If you run `make pyformat`, you can see all the outstanding black formatting changes that should be made; in theory, on a clean checkout, this command shouldn't make any changes, but that's rarely the case. (We should probably be doing a better job verifying this on CircleCI.)